### PR TITLE
Some tweaks and interlinking for code/naming conventions

### PR DESCRIPTION
**Pull request by gewarren**
@BillWagner I noticed a few things that I thought could be improved as I was reading through these.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/fundamentals/coding-style/coding-conventions.md](https://github.com/dotnet/docs/blob/c1e8c66b8d69d01d24ea65e5dfce698a00fa7f1b/docs/csharp/fundamentals/coding-style/coding-conventions.md) | [Common C# code conventions](https://review.learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions?branch=pr-en-us-37342) |
| [docs/csharp/fundamentals/coding-style/identifier-names.md](https://github.com/dotnet/docs/blob/c1e8c66b8d69d01d24ea65e5dfce698a00fa7f1b/docs/csharp/fundamentals/coding-style/identifier-names.md) | ["C# identifier names - rules and conventions"](https://review.learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names?branch=pr-en-us-37342) |
| [docs/fundamentals/code-analysis/style-rules/naming-rules.md](https://github.com/dotnet/docs/blob/c1e8c66b8d69d01d24ea65e5dfce698a00fa7f1b/docs/fundamentals/code-analysis/style-rules/naming-rules.md) | [Code-style naming rules](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/naming-rules?branch=pr-en-us-37342) |

### CODE REVIEW

File: docs/csharp/fundamentals/coding-style/identifier-names.md [hunk 1]
```diff
@@ -20,34 +20,33 @@ For a complete definition of valid identifiers, see the [Identifiers article in
...
 - Use meaningful and descriptive names for variables, methods, and classes.
-- Avoid using single-letter names, except for simple loop counters. See exceptions for syntax examples noted in the following section.
 - Prefer clarity over brevity.
 - Use PascalCase for class names and method names.
 - Use camelCase for method arguments, local variables, and private fields.
 - Use PascalCase for constant names, both fields and local constants.
 - Private instance fields start with an underscore (`_`).
-- Static fields start with `s_`. Note that this isn't the default Visual Studio behavior, nor part of the [Framework design guidelines](../../../standard/design-guidelines/names-of-type-members.md#names-of-fields), but is configurable in editorconfig.
+- Static fields start with `s_`. Note that this isn't the default Visual Studio behavior, nor part of the [Framework design guidelines](../../../standard/design-guidelines/names-of-type-members.md#names-of-fields), but is [configurable in editorconfig](../../../fundamentals/code-analysis/style-rules/naming-rules.md).
```
Comment (gewarren) [inline docs/csharp/fundamentals/coding-style/identifier-names.md:35]: I moved this down so the exceptions could follow it immediately.

File: docs/csharp/fundamentals/coding-style/identifier-names.md [hunk 2]
```diff
@@ -20,34 +20,33 @@ For a complete definition of valid identifiers, see the [Identifiers article in
...
+In addition to the rules, conventions for identifier names are used throughout the .NET APIs. By convention, C# programs use `PascalCase` for type names, namespaces, and all public members. In addition, the `dotnet/docs` team uses the following conventions, adopted from the [.NET Runtime team's coding style](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md):
 
 - Interface names start with a capital `I`.
 - Attribute types end with the word `Attribute`.
 - Enum types use a singular noun for nonflags, and a plural noun for flags.
 - Identifiers shouldn't contain two consecutive underscore (`_`) characters. Those names are reserved for compiler-generated identifiers.
 - Use meaningful and descriptive names for variables, methods, and classes.
-- Avoid using single-letter names, except for simple loop counters. See exceptions for syntax examples noted in the following section.
```
Comment (gewarren) [inline docs/csharp/fundamentals/coding-style/identifier-names.md:30]: I moved this down so the exceptions could follow it immediately.

File: docs/csharp/fundamentals/coding-style/identifier-names.md [hunk 3]
```diff
@@ -20,34 +20,33 @@ For a complete definition of valid identifiers, see the [Identifiers article in
...
+  - Use `v` for variables, `p` for parameters.
+  - Use `r` for `ref` parameters.
 
-- Use `S` for structs, `C` for classes.
-- Use `M` for methods.
-- Use `v` for variables, `p` for parameters.
-- Use `r` for `ref` parameters.
-
-The preceding single-letter names are allowed only in the language reference section.
+> [!TIP]
```
Comment (gewarren) [inline docs/csharp/fundamentals/coding-style/identifier-names.md:46]: Add this tip and link to the code analysis naming rules.

**Review by BillWagner — APPROVED** [review]
Thanks @gewarren 

This looks great, and I'll :shipit: now.

**Pull request merged**